workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -3,6 +3,7 @@
 using Improbable.Worker;
 using Unity.Entities;
 using UnityEngine;
+using Object = System.Object;
```

Comment:
why not just use `object`

Reply:
Ah, this using is no longer necessary, I was using `Object.ReferenceEquals` but I converted them to `ReferenceEquals` instead